tests/gnrc_netif/Makefile

```diff
@@ -27,6 +27,8 @@ CFLAGS += -DGNRC_NETIF_GROUPS_NUMOF=8
 CFLAGS += -DLOG_LEVEL=LOG_NONE
 CFLAGS += -DTEST_SUITES
 
+TEST_ON_CI_WHITELIST += all
```

Comment:
Maybe, so we get around the synchronization issue for now use `native` instead of `all`? :stuck_out_tongue:

Reply:
With your https://github.com/RIOT-OS/RIOT/pull/9672 PR I do not have any synchronization issue anymore on all the boards I tested. https://github.com/RIOT-OS/RIOT/pull/9675#issuecomment-409729428

Reply:
I merged your PR to test with murdock.

Reply:
Interesting... I wasn't able to on IoT-LAB's `samr21-xpro`. Let me retry with a local `samr21-xpro`